paperpi/plugins/home_assistant/home_assistant.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# coding: utf-8
+
+try:
+    from . import layout
+    from . import constants
+except ImportError:
+    import layout
+    import constants
+    
+from pathlib import Path
+from requests import get
+import logging
+
+logger = logging.getLogger(__name__)
+
+def callHome(url, apiToken):
+    headers = {
+        "Authorization": f'Bearer {apiToken}',
+        "content-type": "application/json",
+    }
+
+    logging.debug(f'calling: {url}')
+    response = get(url, headers=headers)
+    logging.debug(f'response: {response.text}')
+    
```

Comment:
something along the lines of:
```
from requests import exceptions as req_exceptions

try:
    response = get(url, headers=headers)
except req_exception.RequestException as e:
    logger.error('failed to callHome due to error {e})
    entity_json = {} # or something along these lines
```

Reply:
New PR in that should make it a bit more resistant

Reply:
I'll try to get to it tomorrow or Wednesday. Thanks for the updates!